application-collabora-ui/src/main/resources/Collabora/Code/UI.xml

```diff
@@ -190,23 +190,21 @@ require(['jquery', 'xwiki-l10n!collabora-attachment'],function($, l10n) {
     return editURL;
   };
 
-  var getCollaboraButton = function(thisButton, fileName, accessRights) {
+  var populateCollaboraButton = function(thisButton, fileName, accessRights) {
     const title = l10n.get(accessRights == 'edit' ? 'edit.title' : 'view.title');
     thisButton.attr('title', title);
     thisButton.find('img').attr('alt', title);
 
     thisButton.attr('href', getActionURL(fileName, accessRights, thisButton.data('editUrl')));
-    return thisButton;
   };
 
   var decorateWithCollaboraButtons = function() {
     // Attachments are displayed using a livetable since 14.6, and using a livedata from 14.8.
     const hasPagination = $("#_attachments table").length &gt; 0;
```

Comment:
Is this still needed?

Reply:
If you're referring to the `populateCollaboraButton` function, yes, as it adds the right title, image alternative and URL.

Reply:
I was referring to the `hasPagination` variable